libnavigation-base/src/test/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectMapperTest.kt

```diff
@@ -232,7 +232,8 @@ class RoadObjectMapperTest {
             Date(60),
             Date(80),
             Date(40),
-            null,
+            null, // TODO country codes
+            null, // TODO country codes
```

Comment:
cc @korshaknn

Reply:
just a reminder, all tests in this class are ignored and can't be fixed right now.
check https://github.com/mapbox/mapbox-navigation-android/issues/4297